apps/back/config/packages/doctrine.yaml

```
@@ -0,0 +1,44 @@
+doctrine:
+    dbal:
+        url: '%env(resolve:DATABASE_URL)%'
+
```

Cf. le commentaire sur le `docker-compose.yaml`

```suggestion
doctrine:
    dbal:
        driver: pdo_mysql
        host: "%env(DB_HOST)%"
        port: "%env(DB_PORT)%"
        dbname: "%env(DB_NAME)%"
        user: "%env(DB_USER)%"
        password: "%env(DB_PASSWORD)%"
        server_version: "%env(DB_VERSION)%"

```